app/views/users/show.html.erb

```diff
@@ -24,15 +24,29 @@
     }
     <% end %>
   </style>
-  <div class="user-profile-header" style="<%= user_colors_style(@user) %>" itemscope itemtype="http://schema.org/Person" itemprop="mainEntity">
-    <meta itemprop="url" content="https://dev.to/<%= @user.username %>">
+
+<script type="application/ld+json">
+  {
+    "@context": "http://schema.org",
```

Comment:
I think we have a bunch of other possibly valuable info that could fit a person object similar to what we have here...

https://jsonld.com/person/

For example, if someone has a public email, we could have that, if they have a job title listed we could use those.

Looks like we could also get `sameAs` info by listing social profile links.

Reply:
Okay, I've made some adjustments to include more information for the Person object. I attempted to add further information including employer name and url as well as a User's education, but haven't been able to get it to work/render without an issue yet--I will continue trying to build this information out. 😊

Reply:
Great. I think what we have looks good and we can make some of those additional fields a follow-on sprint.

One thing though, I think we should probably not include the key at all if it's not present.

This will look kind of messy, but I think something like...

```
  <% if @user.employment_title.present? %>
     "jobTitle": "<%= @user.employment_title %>",
  <% end %>
```
on any column that is optional.

Let's also not leak the user's email if they haven't made it public. We have a column for that, so so like...

```
  <% if @user.email_public %>
     "email": "<%= @user.email %>",
  <% end %>
```